manifests/profile/learning/ssh.pp

```diff
@@ -9,9 +9,9 @@
   augeas { "disable_key_checking":
     context => '/files/etc/ssh/ssh_config',
     changes =>
-      ["set Host[.='*.puppet.vm'] *.puppet.vm",
-       "set Host[.='*.puppet.vm']/StrictHostKeyChecking no",
-       "set Host[.='*.puppet.vm']/UserKnownHostsFile /dev/null"],
+      ["set Host[.='*.puppet.vm *.auroch.vm *.beauvine.vm']*.puppet.vm *.auroch.vm *.beauvine.vm",
```

Comment:
@kjhenner Does there need to be a space after the `]` on this line like there was in the prior version of the file?